packages/_site/src/pages/components/skeleton/index.tsx

```diff
@@ -108,6 +122,86 @@ export default function SelectExample() {
 				height: "12rem",
 			}}
 		/>
+        <Skeleton
+            style={{
+                marginTop: "1rem",
+                width: "20rem",
+                height: "1rem",
+            }}
+        />
+        <Skeleton
+            style={{
+                marginTop: "1rem",
+                width: "16rem",
+                height: "1rem",
+            }}
+        />
+	);
+}
+`}
+                                />
+                            </div>
+                        </div>
+                        <div>
+                            <ExampleBlock
+                                title="Circle"
+                                description="Display circular skeleton, placeholder profile image or icon."
+                            />
+                            <div {...stylex.props(styles.codeWrapper)}>
+                                <PreviewBlock>
+                                    <Skeleton
+                                        style={{
+                                            width: "4rem",
+                                            height: "4rem",
+                                            borderRadius: "50%",
+                                        }}
+                                    />
+                                    <Skeleton
+                                        style={{
+                                            marginTop: "1rem",
+                                            width: "20rem",
+                                            height: "1rem",
+                                        }}
+                                    />
+                                    <Skeleton
+                                        style={{
+                                            marginTop: "1rem",
+                                            width: "16rem",
+                                            height: "1rem",
+                                        }}
+                                    />
```

Comment:
Same as above we don't need multiple different shaped skeleton elements in one example reference block.